src/components/agent-list/ResultNotFound/result.styles.js

```diff
@@ -11,8 +11,11 @@ export const ResultContainer = styled.div`
 `;
 
 export const TextContainer = styled.div`
-  color: ${colors.dustyGray};
+  color: ${colors.mineShaft};
   padding: 2rem 2rem 4rem 2rem;
   text-align: center;
-  width: 100%;
+  width: 80%;
+  font-style:Montserrat;
+  font-size:32px;
```

Comment:
```suggestion
  font-size:32px;
```

we use rem instead of px , please change this to rem instead , 
looks like some one used a div to hold a only a text so can you please fix and use a heading instead of a div.